forge.config.ts

```diff
@@ -0,0 +1,124 @@
+import fs from 'fs-extra';
+import httpProxy from 'http-proxy';
+import http from 'node:http';
+import os from 'node:os';
+import path from 'node:path';
+
+import type { ForgeConfig } from '@electron-forge/shared-types';
+import { MakerDeb } from '@electron-forge/maker-deb';
+import { MakerSquirrel } from '@electron-forge/maker-squirrel';
+import { MakerRpm } from '@electron-forge/maker-rpm';
+import { MakerZIP } from '@electron-forge/maker-zip';
+import { AutoUnpackNativesPlugin } from '@electron-forge/plugin-auto-unpack-natives';
+import { PublisherGithub } from '@electron-forge/publisher-github';
+
+import { version } from './package.json';
+
+const iconDir = path.join(__dirname, 'static/img');
+
+let server: http.Server;
+const PORT = 37492;
+
+const options: ForgeConfig = {
+  hooks: {
+    generateAssets: require('./tools/generateAssets'),
+    preMake: async () => {
+      let dir: string | undefined = undefined;
+      try {
+        const timestampProxiedProxy = httpProxy.createProxyServer({});
+
+        server = http.createServer((req, res) => {
+          return timestampProxiedProxy.web(req, res, {
+            target: 'http://timestamp.digicert.com',
+          });
+        });
+
+        await new Promise((resolve) => {
+          server.listen(PORT, () => {
+            resolve(null);
+          });
+          console.log(`server listening on port ${PORT}`);
+        });
+
+        dir = await fs.mkdtemp(
+          path.resolve(os.tmpdir(), 'slack-builder-folder-'),
+        );
+      } finally {
+        if (dir) await fs.remove(dir);
+      }
+    },
+    postMake: async () => {
+      server.close();
+      console.log(`server closing`);
+      // FIXME: we should be able to just void return the postMake hook upstream?
```

Comment:
I have a PR for this upstream in Forge so we can remove this once we land it in the next release.